src/Quiz.js

```diff
@@ -0,0 +1,45 @@
+import React, { Component } from "react";
+import IntervalExercise  from './IntervalExercise.js'
+import PropTypes from "prop-types";
+
+function ContinueButton({ onClick }) {
+    return (
+        <button onClick={onClick}> Continue </button>
+    )
+}
+
+class Quiz extends Component {
+    render() {
+        var maybeContinueButton;
+        if (// we've answered correctly
+            this.props.submittedAnswers.includes(this.props.interval.toString())
+            // and this is not the last exercise
+            && this.props.currentExercise < this.props.numExercises - 1
+          ) {
+          maybeContinueButton = <ContinueButton
+            onClick={this.props.onContinueClick}
+          />
+        }
+
+        return (<div>
+            <IntervalExercise
+              interval={this.props.interval}
+              possibleAnswers={this.props.possibleAnswers}
+              submittedAnswers={this.props.submittedAnswers}
+              onAnswerClick={this.props.onAnswerClick}
+            />
+            {maybeContinueButton}
```

Comment:
This is somewhat subjective, but the way I would normally do this would be something more like this. Above:

```javascript
const quizCompleted = (
    // we've answered correctly
    this.props.submittedAnswers.includes(this.props.interval.toString())
    // and this is not the last exercise
    && this.props.currentExercise < this.props.numExercises - 1
)
```

Here:

```suggestion
            {quizCompleted && <ContinueButton onClick={this.props.onContinueClick} />}
```

See the React docs for tips on [conditional rendering](https://reactjs.org/docs/conditional-rendering.html).